### DISCUSSION TITLE
Mono / stereo split internal recording option

### DISCUSSION THREAD

**Discussion by sichtbeton** [SynthstromAudible/DelugeFirmware#654] [Ideas]
would it be possible to add two more recording options (like “Left” etc) which would be like “output” but only one channel? This way you could record  the output of the deluge, but only the left or right channel. This way you can do a stereo / mono split.

**Comment by duncanrmi** [comment]
I think there's a bigger discussion about this to be had in the context of exporting stems; did you mean that, or is this for resampling?

**Comment by m-m-adams** [comment]
Yes to all these questions, it's something I'd like to do once we have the audio engine stable in general
